### PR TITLE
add boston budget site to examples

### PR DESCRIPTION
**Pull request by erquhart**
Source: https://twitter.com/jimafisk/status/859593116927524864

### CODE REVIEW

File: site/data/examples.yaml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+    description: Executive summary of the City of Boston’s fiscal year 2018 recommended budget.
+    link: https://budget.boston.gov
+    thumbnailurl: /img/examples/boston-budget.png
+    tools:
+      - tool: Jekyll
+      - tool: Gulp
+      - tool: Chart.js
   - title: Skcript
-    description: We build Artificial Intelligence powered products for today's businesses.
+    description: We build Artificial Intelligence powered products for today’s businesses.
```
Comment (erquhart) [inline site/data/examples.yaml]: Changed to a proper apostrophe, otherwise syntax highlighting for this file goes sour.

**Comment by erquhart** [comment]
Hmm wrong image, updating...

**Comment by erquhart** [comment]
Good to go.

**Comment by verythorough** [comment]
LGTM!

**Pull request merged**